src/scripts/pokemons/PokemonList.ts

```diff
@@ -14177,7 +14177,7 @@ pokemonList.forEach(p => {
 
     p.attack = Math.max(10, Math.floor(Math.sqrt(baseDefense * baseStamina) * baseOffense / 250));
     if (p.baby) {
-        p.evolutions?.forEach(evo => pokemonDevolutionMap[evo.evolvedPokemon] = evo.basePokemon);
+        p.evolutions?.forEach(evo => pokemonDevolutionMap[evo.getEvolvedPokemon()] = evo.basePokemon);
```

Comment:
This makes `pokemonDevolutionMap` vary depending on what time of day you loaded the game. It isn't going to cause a problem with Eevee, but it could with some hypothetical future pokemon.

Reply:
I've added a comment regarding this. I _think/hope_ this will ever become an issue

Reply:
It will affect chingling and budew IIRC